src/components/dev-popover/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useRef, useState } from 'react'
+import Popover from 'components/popover'
+
+/**
+ * Renders non-interactive children into a button element,
+ * and displays a popover when the button is activated.
+ *
+ * Intended for use with work-in-progress elements.
+ * Should not be used in production.
+ */
+function DevPopover({
+  children,
+  note,
+  title = 'Note',
+}: {
+  children: React.ReactNode
+  note: React.ReactNode
+  title?: string
+}): React.ReactElement {
+  const [showDialog, setShowDialog] = useState(false)
+  const triggerRef = useRef(null)
+  return (
+    <>
+      <button
```

Comment:
Love that a `<button>` is rendered here! Covers some usability cases that might otherwise be tricky to handle. 👏🏻